crates/db_schema/src/impls/actor_language.rs

```diff
@@ -233,6 +237,28 @@ fn update_languages(
   }
 }
 
+/// If all languages are returned, return empty vec instead
+fn convert_read_languages(
+  conn: &mut PgConnection,
+  language_ids: Vec<LanguageId>,
+) -> Result<Vec<LanguageId>, Error> {
+  static ALL_LANGUAGES_COUNT: OnceCell<usize> = OnceCell::new();
```

Comment:
I'm trying to think of others things we could use this for, but can't think of any rn. Almost all our DB data changes, and even the new local site settings stuff I've added probably needs to be live-read.